apps/03TicTacToe.js

```diff
@@ -40,7 +40,12 @@ function checkForWin() {
 }
 
 function ticTacToe(row, column) {
-  // Your code here
+  board[row][column] = playerTurn;
+
+}
+
+function toggleTurn() {
+  playerTurn = (playerTurn === 'X')? 'O' : 'X';
```

Comment:
nitpick time - make sure you put a space before that ternary operator:
`playerTurn = (playerTurn === 'X') ? 'O' : 'X';`

:)